library/src/main/java/io/constructor/data/model/Group.kt

```diff
@@ -1,7 +1,8 @@
 package io.constructor.data.model
 
 import com.squareup.moshi.Json
+import io.constructor.data.model.search.Path
 import java.io.Serializable
 
 
-data class Group(@Json(name="display_name") var displayName: String?, @Json(name="group_id") var groupId: String, @Json(name="path") var path: String? = null) : Serializable
\ No newline at end of file
+data class Group(@Json(name="display_name") var displayName: String?, @Json(name="group_id") var groupId: String, @Json(name="path") var path: String? = null, @Json(name="path_list") var pathList: List<Path>? = null) : Serializable
```

Comment:
Let's rename to `ResultGroup`... to differentiate from the rolled up groups with counts at the top level of a search response?